qemu/tests/cfg/vhost_with_cgroup.cfg

```diff
@@ -1,6 +1,7 @@
 - vhost_with_cgroup:
     virt_test_type = qemu
     only virtio_net
+    only Host_RHEL.m5 Host_RHEL.m6 Host_RHEL.m7 Host_RHEL.m8
```

Comment:
I think we can remove `Host_RHEL.m5 Host_RHEL.m6`. What do you think? cc @yanan-fu